src/client/components/editor/task-list.js

```diff
@@ -10,9 +10,9 @@ const NotificationPanel = require('../notification/panel');
 const getIncompleteEntities = doc => doc.entities().filter(ent => !ent.completed());
 
 
-const eleEvts = [ 'rename', 'redescribe' ];
-const entEvts = [ 'modify', 'associated', 'unassociated', 'complete', 'uncomplete'];
-const intnEvts = [ 'retype' ];
+const eleEvts = [ 'rename' ];
+const entEvts = [ 'complete', 'uncomplete' ];
```

Comment:
Shouldn't these be in eleEvts?  Any ele can be completed.  For interactions, it means the same thing in the UI: A user must complete the wizard to get 'complete'.

Reply:
That may need a separate case for the notification message, as the interaction doesn't have a name() I think.

It does have a toString() though, which describes the interaction concisely, e.g. "A phosphorylates B"

Reply:
For incomplete interactions, though, the toString() may not be very meaningful.  It could even be something like "(?) interacts with (?)", so maybe a different message would be needed

Reply:
I wasn't aware of the changes made a month ago that allowed all elements to have a completed state.  Right now the task list only focuses on incomplete entities, but I can extend it to support interactions too.